pulser-core/pulser/noise_model.py

```diff
@@ -321,6 +399,7 @@ def _find_relevant_params(
                 or nt_ == "detuning"
                 or (nt_ == "amplitude" and amp_sigma != 0.0)
                 or (nt_ == "SPAM" and state_prep_error != 0.0)
+                or (nt_ == "register" and temperature != 0.0)
```

Comment:
Is this still needed now that `"register"` requires `temperature > 0.0`?
Sorry, I know we have been going back and forth on this